packages/sdk/lib/base.ts

```diff
@@ -184,20 +191,24 @@ export class Itly {
 
   /**
    * Initialize the Itly SDK. Call once when your application starts.
-   * @param context Additional context properties to add to all events.
    * @param options Configuration options to initialize the Itly SDK with.
    */
-  load(context?: Properties, options?: Options) {
+  load(options: LoadOptions = {}) {
```

Comment:
I think renaming this parameter `loadOptions` would make the following code a little easier.

E.g.
```
const {
  context,
  ...options
} = loadOptions;
```

Reply:
Done